INSTALL.md

```diff
@@ -0,0 +1,25 @@
+
+## Setup CactusBot:
+```
+git clone https://github.com/cactusdev/cactusbot
```

Comment:
It's not *needed*, but it looks much nicer. (Our Twitter handle isn't `@cactusdevteam`, Discord isn't `cactusdev`, etc.) :)